XDS_PROTOCOL.md

```diff
@@ -98,8 +98,9 @@ messages:
 The version provides Envoy and the management server a shared notion of the
 currently applied configuration, as well as a mechanism to ACK/NACK
 configuration updates. If Envoy had instead rejected configuration update __X__,
-it would reply with its previous version, which in this case was the empty
-initial version:
+it would reply with [`error_detail`] (https://github.com/envoyproxy/data-plane-api/blob/master/envoy/api/v2/discovery.proto#L56) populated and its previous version, which in this case was the empty
```

Comment:
Can you use a GH permalink here? As the referenced proto changes, that line number if going to be out-of-date. I think it might be even better to link to the doc rendering of the proto, since that will give you a stable href link that will track master but not fall out-of-date.

Reply:
@htuch updated with doc link. Please check if that is what you mean by doc link or not?